engine/flink/executor/src/test/scala/pl/touk/nussknacker/engine/process/registrar/InterpretationFunctionFlinkGraphSpec.scala

```diff
@@ -87,7 +93,7 @@ class InterpretationFunctionFlinkGraphSpec extends FlinkStreamGraphSpec {
   }
 
   private def prepareProcess(useAsyncInterpretation: Boolean) = ScenarioBuilder
-    .streaming(scenarioId)
+    .streaming(scenarioName.value)
```

Comment:
Using `ProcessName` value class in every place where `string` is not in the scope of this change.

Reply:
The DSL for scenarios is one of rare places which is used by a human instead of the code. In such a situation value class introduce a cost instead of giving benefits.

Reply:
This DSL is used by developers :) Of course we can support two way of passing argument.. But let's leave it.

Reply:
I thought about that, but we already have 6 combinations of `ScenarioBuilder.streaming`, `ScenarioBuilder.streamingLite` etc. Adding another 6 doesn't look like a good approach :-/